test/e2e/flask/multichain-api/evm/wallet_createSession.spec.ts

```diff
@@ -335,6 +339,11 @@ describe('Multichain API', function () {
             await editConnectedAccountsModal.addNewAccount();
 
             await connectAccountConfirmation.checkPageIsLoaded();
+
+            await connectAccountConfirmation.checkForAccountsInPermissionList([
+              'Account 1',
+              'Account 2',
+            ]);
```

Comment:
to fix this other error appearing in the latest step -- now we ensure both accounts appear in the permissions dialog before confirming it:

<img width="1087" height="110" alt="Image" src="https://github.com/user-attachments/assets/9a4be7a2-cb4f-4657-9bf5-6f63c26f2688" />